Components/Product.js

```diff
@@ -128,11 +129,30 @@ class ProductScreen extends Component {
         }
     }
 
+    _checkAllergies() {
+        const { product, isLoading} = this.state;
+        if (!isLoading) {
+            let user = UserService.findAll()[0];
+            let allergens = [];
+            for (let allergen of product.allergens_ids) {
+                if (Array.from(user.allergies).includes(allergen)) {
+                    allergens.push(allergen); //change to allergen name instead of id
+                }
+            }
+            Alert.alert(
```

Comment:
Alert should only be triggered when allergens is not empty

![img_1268](https://user-images.githubusercontent.com/30070925/51118063-bd3d5480-180f-11e9-95fe-184881acd69e.PNG)

<img width="204" alt="capture d ecran 2019-01-14 a 15 14 34" src="https://user-images.githubusercontent.com/30070925/51118082-c7f7e980-180f-11e9-8e64-8bf9bfbc4e20.png">

![img_1269](https://user-images.githubusercontent.com/30070925/51118093-cfb78e00-180f-11e9-9316-e30f9fc398e2.PNG)

Reply:
done